editor/utils/test/undoable-reducer.js

```diff
@@ -1,11 +1,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { undoable, combineUndoableReducers } from '../undoable-reducer';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.